pkgs/by-name/pl/platformsh/package.nix

```diff
@@ -39,7 +42,10 @@ stdenvNoCC.mkDerivation (finalAttrs: {
   installPhase = ''
     runHook preInstall
 
-    install -Dm755 platformsh $out/bin/platformsh
+    install -Dm755 platform $out/bin/platform
+
+    installShellCompletion completion/bash/platform.bash \
```

Comment:
Wait, can it detect zsh in this case? See https://nixos.org/manual/nixpkgs/unstable/#installshellfiles

Reply:
```
[nix-shell:~/Developer/nixpkgs/result]$ tree
.
├── bin
│   └── platformsh
└── share
    ├── bash-completion
    │   └── completions
    │       └── platform.bash
    └── zsh
        └── site-functions
            └── _platform
```

the result looked fine for me